bottlerocket-agents/src/bin/ec2-resource-agent/ec2_provider.rs

```diff
@@ -149,9 +149,17 @@ impl Create for Ec2Creator {
             );
 
         info!("Starting instances");
-        let instances = run_instances
-            .send()
-            .await
+        let mut run_instance_result = run_instances.clone().send().await;
+        if let Err(SdkError::ServiceError { err, raw: _ }) = &run_instance_result {
+            if matches!(&err.code(), Some("InvalidParameterValue")) {
+                info!("An error occured while trying to run instances. Will retry in 60s.");
+                tokio::time::sleep(Duration::from_secs(60)).await;
```

Comment:
Does 10s seem reasonable? Maybe we don't need the full 60?

Reply:
Aws says it can take up to 6 mins for propagation to complete. I haven't had any failures with 60s, so I went with that.